src/pages/view-test-result/cat-a-mod1/view-test-result.cat-a-mod1.page.ts

```diff
@@ -78,11 +77,10 @@ export class ViewTestResultCatAMod1Page extends BasePageComponent implements OnI
       .getTestResult(this.applicationReference, this.authenticationProvider.getEmployeeId())
       .pipe(
         map((response: HttpResponse<any>): string => response.body),
-        // todo: PREP-AMOD1 change to CatAMod1UniqueTypes when schema changes are ready
-        map(data => this.testResult = this.compressionProvider.extractTestResult(data) as CatBEUniqueTypes.TestResult),
+        map(data => this.testResult = this.compressionProvider.extractTestResult(data) as TestResultCatAM1Schema),
         tap(() => this.handleLoadingUI(false)),
         catchError((err) => {
-          this.testResult = categoryBETestResultMock;
+          this.testResult = categoryAM1TestResultMock;
```

Comment:
if an error is thrown and caught at this point, won't it mean that the live application will display this mock data?

Reply:
was existing code - I'll raise with @ammarhaiderbjssm, but agree. prob shouldn't be there.

Reply:
Can we just get rid of this assignment from B+E and AMod1 and test to see what happens?

Reply:
will look separately from the other comments.